python/py-eric-ide/Portfile

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-eric-ide
+set real_name       eric6
+version             19.04
+revision            0
+
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             GPL-3+
+maintainers         {gmail.com:giansalvo.gusinu @giansalvo} openmaintainer
```

Comment:
Has @giansalvo agreed to maintain this port? If not, please use `nomaintainer` unless you are willing to maintain this.

Reply:
I don't know... waiting for his response. If he doesn't answer in the next day or so, I'll remove him and use ```nomaintainer```.

Could you please try and install ```py27-eric-ide``` and see if it starts correctly for you? In [Trac ticket 56636](https://trac.macports.org/ticket/56636) Joseph says that for him it doesn't start, but I cannot reproduce the problem and suspect it is something in his setup...

Reply:
@reneeotten  please remove me as maintainer of this port